src/components/auth/login/LoginForm.tsx

```diff
@@ -25,85 +24,65 @@ const validationSchema: yup.SchemaOf<LoginFormData> = yup
   .shape({
     email: yup.string().email().required(),
     password: yup.string().min(6, customValidators.passwordMin), // .required(),
-    csrfToken: yup.string(), // .required(),
   })
 
 const defaults: LoginFormData = {
   email: '',
   password: '',
-  csrfToken: '',
 }
 
 export type LoginFormProps = {
   initialValues?: LoginFormData
-  csrfToken: string
 }
```

Comment:
Maybe we can have a `returnUrl` on the login form that you pass and when the user logs in it returns him there. By default it can be the `baseUrl` or `undefined`

Reply:
That can be a seperate issue though for the LoginForm as a whole!

Reply:
There's a query param `callbackUrl` which is handled after login

http://localhost:3040/login?callbackUrl=%2Fadmin

On `LoginForm` we can add a `onLogin` prop which will handle the callback with a default redirect

Let's do that in a separate issue/PR